medusa/post_processor.py

```diff
@@ -825,12 +828,34 @@ def _priority_from_history(self, show_id, season, episodes, quality):
                         if self.file_name != download_name:
                             self.in_history = True
                             return
+
                     else:
                         # There aren't any other files processed before for this
                         # episode and quality, we can safely say we want this file
                         self.in_history = True
                         return
 
+    def _is_priority(self, ep_obj, old_ep_quality, new_ep_quality):
+            """
+            Determine if the episode is a priority download or not (if it is expected).
+
+            Episodes which are expected (snatched) or larger than the existing episode are priority, others are not.
```

Comment:
This doesn't exactly hold true as you can have a larger HDTV than a WEB-DL, I get this a LOT with CW shows.